app/src/main/java/com/simplecity/amp_library/playback/MultiPlayer.java

```diff
@@ -184,17 +183,7 @@ void setVolume(float vol) {
     }
 
     int getAudioSessionId() {
-        if (Build.VERSION.SDK_INT > 8) {
-            int sessionId = 0;
-            try {
-                sessionId = mCurrentMediaPlayer.getAudioSessionId();
-            } catch (IllegalStateException ignored) {
-                //Nothing to do
-            }
-            return sessionId;
-        } else {
-            return 0;
-        }
+        return mCurrentMediaPlayer.getAudioSessionId();
```

Comment:
The try-catch should remain in-tact here. It's not related to the SDK version - there's a chance the `MediaPlayer` is in a state where `getAudioSessionId()` will throw an `IllegalStateException`.